src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -459,6 +459,30 @@ final class VariablesContainerTest: XCTestCase {
         XCTAssertFalse(container.isProjectVariableOrList(list2))
     }
 
+    func testIsProjectVariable() {
+        let userVariable1 = UserVariable(name: "testVar1", isList: false)
+        let userVariable2 = UserVariable(name: "testVar2", isList: false)
+
+        let container = VariablesContainer()
+
+        container.programVariableList.add(userVariable1 as Any)
+
+        XCTAssertTrue(container.isProjectVariable(userVariable1))
+        XCTAssertFalse(container.isProjectVariable(userVariable2))
+    }
+
+    func testIsProjectList() {
+        let list1 = UserList(name: "testList1")
+        let list2 = UserList(name: "testList2")
```

Comment:
Please add `list2` as an object variable for a object to distinguish between object and program variables (that's the purpose of the function you are testing).